hogben/utils.py

```diff
@@ -200,8 +200,10 @@ def fisher(qs, xi, counts, models, step=0.005):
             lb = np.array([param.bounds.limits[0] for param in xi])
             ub = np.array([param.bounds.limits[1] for param in xi])
 
+        if importance is None: # Default importance equals one for each param
+            importance = np.ones(len(lb))
```

Comment:
We might have to think a little about how we want to implement this, as writing this in currently requires the user to know the order of parameters in `xi`. Possibly it isn't something to be dealt with now though

Reply:
This should definitely be a thing to consider. And the more I think about it, the less I'm convinced that we want this to be order-dependent, as it is something that quite easily can go wrong.

From a user-facing perspective, it may be easiest if you attach the importance to the parameter directly. So that you can just say I want the `layer1.rough` parameter to be weighted by 2, which than easily can assigned using layer1.rough.importance = 2. 

However, I am not 100% sure about assigning attributes to classes that are not our own. Code-wise it works fine (I tested), but purely theoretically it could be possible that refnx/refl1d would add an attribute with the same name in the future that we would be overwriting. Probably very unlikely, but doing it anyway smells.

A roundabout way would be to do it with a dictionary.  Like the default for refnx would be `importance = {'layer1.rough': 1, 'layer2.rough': 1, 'layer1.thick': 1} ` (or just set the values to 1 on runtime in case they're missing). And the attributes can be set like` importance['layer1.rough'] `= 2.  But that doesn't feel very elegant either.

Either way, I'll wait for your opinions on this particular point before formally requesting a new code review on this PR. As I personally do feel like an order-independent approach (e.g. by attaching the values to the parameters) would definitely be much better and is much less prone to user-error.